x/orderbook/model.go

```diff
@@ -69,28 +69,31 @@ func (o *OrderBook) Copy() orm.CloneableData {
 
 // Validate is always succesful
 func (o *OrderBook) Validate() error {
-	// if err := o.Metadata.Validate(); err != nil {
-	// 	return err
-	// }
-	if err := isGenID(o.ID, true); err != nil {
-		return err
-	}
-	if err := isGenID(o.MarketID, false); err != nil {
-		return errors.Wrap(err, "market id")
-	}
+	var errs error
+
+	//errs = errors.AppendField(errs, "Metadata", o.Metadata.Validate())
+	errs = errors.AppendField(errs, "ID", isGenID(o.ID, true))
+	errs = errors.AppendField(errs, "MarketID", isGenID(o.MarketID, false))
+
 	if !coin.IsCC(o.AskTicker) {
-		return errors.Wrap(errors.ErrModel, "invalid ask ticker")
+		errs = errors.Append(errs,
+			errors.Field("AskTicker", errors.ErrCurrency, fmt.Sprintf("Invalid ask ticker: %s", o.AskTicker)))
```

Comment:
Please keep the error string lowercase. It is not a sentence.
https://github.com/golang/go/wiki/CodeReviewComments#error-strings